cumulus/steps/dev_tools/code_build_action.py

```diff
@@ -49,10 +49,10 @@ def __init__(self,
     def handle(self, chain_context):
 
         print("Adding action %s Stage." % self.action_name)
-        suffix = "%s%s" % (self.stage_name_to_add, self.action_name)
+        full_action_name = "%s%s" % (self.stage_name_to_add, self.action_name)
 
-        policy_name = "CodeBuildPolicy%s" % chain_context.instance_name
-        role_name = "CodeBuildRole%s" % suffix
+        policy_name = "%sCodeBuildPolicy" % chain_context.instance_name
```

Comment:
this isn't a managed policy so we won't see the name in AWS, but I guess it asks the question if consistency even here would be helpful?  Suffix vs prefix...